include/swift/Reflection/RuntimeInternals.h

```diff
@@ -92,7 +92,8 @@ struct StackAllocator {
 
 template <typename Runtime>
 struct AsyncTaskPrivateStorage {
-  typename Runtime::StoredSize Status;
+  typename Runtime::StoredSize Status1;
```

Comment:
If I'm reading the code correctly, this corresponds to the `TaskStatusRecord` pointer. Do we want it to be a `Runtime::StoredPointer` instead of a `Runtime::StoredSize`?

Reply:
Yeah, it doesn't really matter much since we just need to get the sizes right, but marking this as a pointer would be the right thing to do. Breaking this out into a separate `ActiveTaskStatus` thing would also be sensible. I'll do both.